set-definitions/src/main/java/eu/europeana/set/definitions/model/vocabulary/WebUserSetFields.java

```diff
@@ -44,6 +44,7 @@
     public final static String IDENTIFIER = "identifier";
     public final static String TITLE = "title";
     public final static String DESCRIPTION = "description";
+    public final static String DC_DESCRIPTION = "dcDescription";
```

Comment:
This field must be removed. dc:decription meand simply description in the JsonLd serialization

Reply:
Field removed